components/nav/Nav.tsx

```diff
@@ -2,16 +2,29 @@ import Image from 'next/image';
 import Logo from '@/components/assets/logo.svg';
 import ProfileDropdown from '@/components/nav/ProfileDropdown';
 import Link from 'next/link';
+import { headers } from 'next/headers';
+
+export default async function NavBar() {
+  const headersList = headers();
+  const referer = headersList.get('referer');
+  let url = '/private-cloud/products/all';
+  if (typeof referer === 'string') {
```

Comment:
let's use `import _isString from 'lodash-es/isString` for better readability.

Reply:
changed to using _isString